src/styles/blocks/header.scss

```diff
@@ -1,3 +1,5 @@
+$hover-text: #34568b;
```

Comment:
Same. Btw, there should be another color for links
![image](https://user-images.githubusercontent.com/96409563/202647003-44e7a22d-0626-402e-89fa-7a4fbeb9b799.png)